src/dialer/index.js

```diff
@@ -63,18 +63,18 @@ class Dialer {
     transportManager,
     peerStore,
     addressSorter = publicAddressesFirst,
-    concurrency = MAX_PARALLEL_DIALS,
-    timeout = DIAL_TIMEOUT,
-    perPeerLimit = MAX_PER_PEER_DIALS,
+    maxParallelDials = MAX_PARALLEL_DIALS,
```

Comment:
This is an internal breaking change. No one should be creating a dialer from here though. If we don't want to change this, we can create a separate typedef for core options and keep things as they were. However, keeping different names for the same things does not seem the best call

Reply:
_"No one should be"_ famous last words, how much confidence do you have in this?

Reply:
Does not make sense to require the `Dialer` and use it as a specific piece. However, I agree and it will be safe to just ship a breaking change at this point

Reply:
If you're adding types it's good to ship as a major anyway as there will almost certainly be some disruption.